templates/includes/search_form.html

```diff
@@ -1,7 +1,7 @@
 {% load core_tags %}
 
 {% comment %} 
-  This form will appear in more than one place on the page (header (though that's a tsx version), footer, and possibly elsewhere).
+  This form will appear in more than one place on the page (main nav (though that's a tsx version), footer, and on the search page).
```

Comment:
ah, gotcha. I was reading it as it being the same `uniqueID` passed to both template and tsx and was kinda confused. Makes more sense now!